docs/en_us/internal/testing.rst

```diff
@@ -122,7 +122,7 @@ This includes python, javascript, and documentation tests. It does not,
 however, run any acceptance tests.
 
 Note -
```

Comment:
@vkaracic as an FYI, a nifty feature to use for a note is the ```.. note::``` format. When rendered into HTML, that puts the entire following paragraph into a colored box so that it stands out.
```.. note:: paver is a scripting tool...```